### PR TITLE
Prepare propagation services for SRR

### PR DESCRIPTION
**Pull request by jdries**

### CODE REVIEW

File: propagation/intercomparison.md [hunk 1]
```diff
@@ -6,3 +6,24 @@ to choose.
...
+- A custom implementation on the Sentinelhub, exposed via the Terrascope openEO backend.
+
+All three implementations implement sar_backscatter, but may support different parameters. Hence the range of supported
+parameters is an important point of comparision, but also the cost in platform credits as well as correctness of the results
+can be easily compared.
+
+## Call for service providers
+
+We are looking for a first category of algorithms to be intercompared. If you have an algorithm, consider onboarding it
+via the [Toolbox cloudification](./toolboxcloud.md) service, and request your peers to do the same so a first
```
Comment (JanssenBrm) [inline propagation/intercomparison.md:25]: This probably has to be reviewed based on discussion going on in https://github.com/ESA-APEx/apex_documentation/pull/17

**Review by JanssenBrm — APPROVED** [review]
Thanks for the text Jeroen! I have made a couple of changes, but feel free to revert them if you don't agree. 

**Intercomparison**
- Removed the "we" from the text to be more consistent with other parts of the document. 
- Moved the backscatter to a dedicated example section. This breaks the larger amount of text and provides a clearer context for the reader.
- Transformed the bullet list of the backscatter implementations into a table to make it more clear towards readers.


**Upscaling**
- Reordered the text a bit to first explain what the service is before placing a CTA.
- Added some tooltips to make some important messages stand out.
- Added an example tool (openEO batch job manager) to describe what tools we make available. Feel free to amend or add features.

File: propagation/upscaling.md [hunk 2]
```diff
@@ -5,4 +5,41 @@ over larger geographic areas. The service assumes that the algorithm is already
...
+As upscaling to large areas can be challenging, APEx offers the possibility to request time-limited support for your 
+ESA project. Based on your needs, APEx can provide:
+
+- Basic training in the use of the upscaling service.
+- Definition of an appropriate upscaling strategy, tailored to your needs.
+- Selection of the most appropriate platform for your upscaling task.
+
+In order to make use of the upscaling service, please be aware that careful planning is needed:
+ 
+ - Your service needs to be onboarded in APEx before upscaling can start. Please refer to our [Algorithm Hosting](./hosting.md) service for more information.
```
Comment (JanssenBrm) [inline propagation/upscaling.md:29]: Would this be a "hard" requirement? Let's say the algorithm is still under development and they would like to see how it compares to others. Would those algorithms be onboarded first, but hidden in the catalogue? An alternative option could be that we say that the algorithm needs to be in line with the interoperability and compliance guidelines to ensure we get a good base algorithm to work with. What do you think?

File: propagation/intercomparison.md (hunk 1 above)
Comment (jdries) [reply propagation/intercomparison.md]: yes, but not a blocker to merge this one already?

**Comment by jdries** [comment]
@JanssenBrm  if no major objections, I suggest we merge this one, also allowing to prepare the deliverable.

**Pull request merged**